### ISSUE TITLE
x/tools/cmd/present: playground output is racy, brought out by new scheduler

### ISSUE THREAD

**Issue by josharian** [golang/go#11534]
race.slide:

```
Demo

* Hello

.play race.go
```

race.go:

```go
package main

import "fmt"

func main() {
	fmt.Println("hello, race")
}
```

Compile present using Go 1.4, open the demo, and hit Run repeatedly. Result: "hello, race" displays consistently.

Compile present using Go tip, open the demo, and hit Run repeatedly. Result: "hello, race" displays maybe one time in three, although the program claims to have exited successfully.

This is probably a logical race in present or playground that the scheduler changes have exposed.

This is probably a high priority, in that I wouldn't want any GopherCon presenters to do a last minute `present` upgrade and have any unpleasant discoveries.

**Comment by adg** [comment]
There was a comment saying "potential race here" right about the culprit
line. Yay past-me?

**Comment by adg** [comment]
Fixed as part of fixing #11507
